### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by pittengermd**
Added TFSF info

### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

**Comment by pittengermd** [comment]
Added TFSF info to registration.json

File: registry.json [hunk 1]
```diff
@@ -4832,5 +4832,19 @@
...
+    "ticker": "TFSF",                                                       
+    "poolId": "0ca8e61d6ce513468766d3a8e5dc04ac111642a09339b4562563c3f3", 
+    "social": {                                                            
+      "twitter": "thefreecrypto",
+      "website": "https://www.thefreecrypto.com",
+      "facebook": "thefreespeechforum",  
+      "instagram": "thefreespeechforum",
+      "youtube": "thefreespeechforum.com"
+    },
+    "telegram": "freespeechforum",
```
Comment (Bez625) [inline registry.json:4846]: The value for this should be the username of an individual and not a group or channel. Please change

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
Please address the comments or I will look to close in the coming days. Thanks

**Comment by Bez625** [comment]
Closing as no updates. Please feel free to raise a new PR if you still wish to join

**Pull request closed without merging**